int32 ant_count